openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -351,7 +352,7 @@ def _gradfunc(self, dv_dict, func_dict):
         """
 
         fail = 1
-        sens_dict = {}
+        sens_dict = OrderedDict() #{}
```

Comment:
unnecessary OrderedDict. It should also be moved to the end of the except branch of the try/except block, since it isn't needed unless the call to self.calc_gradient fails.